doc/api/buffer.md

```diff
@@ -38,11 +38,11 @@ const buf3 = Buffer.allocUnsafe(10);
 // Creates a Buffer containing [0x1, 0x2, 0x3].
 const buf4 = Buffer.from([1, 2, 3]);
 
-// Creates a Buffer containing ASCII bytes [0x74, 0x65, 0x73, 0x74].
-const buf5 = Buffer.from('test');
```

Comment:
My reasoning was that it's more consistent with the other examples in the doc, and also shows the difference between the two encodings that will affect the output (`'test'` is the same in ascii and utf8 but not `'tést'`). That may or may not be important

Just changing the text to ASCII/UTF-8 is still an improvement but I think this example is clearer.